extensions/frontend-tiles/src/integration-test/IntegrationTest.test.ts

```diff
@@ -0,0 +1,108 @@
+import { IModelApp } from "@itwin/core-frontend";
+import { expect } from "chai";
+import { TestContext } from "./TestContext";
+import { GraphicRepresentationStatus } from "../GraphicsProvider/GraphicRepresentationProvider";
+
+async function getExport(exportId: string, accessToken: string, urlPrefix?: string): Promise<any> {
+  const headers = {
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    Authorization: accessToken,
+    /* eslint-disable-next-line @typescript-eslint/naming-convention */
+    Accept: "application/vnd.bentley.itwin-platform.v1+json",
+  };
+
+  // obtain export for specified export id
+  const url = `https://${urlPrefix}api.bentley.com/mesh-export/${exportId}`;
+  try {
+    const response = await fetch(url, { headers });
+    const result = await response.json();
+    return result;
+  } catch (err) {
+    return undefined;
+  }
+}
+
+describe("MeshExportServiceIntegrationTest", () => {
+  let accessToken: any;
+  let urlPrefix: any;
+  let exportId: any;
+
+  before(async () => {
+
+    // obtain access token
+    const testContext = await TestContext.instance();
+    accessToken = testContext.getAccessToken();
+
+    const iModelId = process.env.MES_IMODEL_ID || "";
+    urlPrefix = process.env.imjs_url_prefix || "";
```

Comment:
inconsistent casing

Reply:
imo, you went the wrong way.
all the env vars should be capitalized. this is the standard across the entire monorepo, why start a new trend?